examples/parallel_sddp.jl

```diff
@@ -0,0 +1,57 @@
+
+import StochDynamicProgramming
+
+
+"""
+Solve SDDP in parallel.
+
+# Arguments
+* `model::SPmodel`:
+    the stochastic problem we want to optimize
+* `param::SDDPparameters`:
+    the parameters of the SDDP algorithm
+* `V::Array{PolyhedralFunction}`:
+    the current estimation of Bellman's functions
+* `n_parallel_pass::Int`: default is 4
+    Number of parallel pass to compute
+* `synchronize::Int`: default is 5
+    Set when to synchronize the cuts between the different processes.
+* `display::Int`: default is 0
+    Says whether to display results or not
+
+# Return
+* `V::Array{PolyhedralFunction}`:
+    the collection of approximation of the bellman functions
+"""
+function psolve_sddp(model, params, V; n_parallel_pass=4,
+                     synchronize=5, display=0)
+    # Redefine seeds in every processes to maximize randomness:
+    @everywhere srand()
```

Comment:
You should do something like `srand(myid())` so the seed is different for each processor.